lib/Server.js

```diff
@@ -570,8 +570,21 @@ class Server {
         const value = options.https[property];
         const isBuffer = value instanceof Buffer;
 
-        if (value && !isBuffer && fs.lstatSync(value).isFile()) {
-          options.https[property] = fs.readFileSync(path.resolve(value));
+        if (value && !isBuffer) {
+          let stats = null;
+
+          try {
+            stats = fs.lstatSync(value).isFile();
+          } catch (error) {
+            // ignore error
+          }
+
+          if (stats) {
+            // It is file
+            options.https[property] = fs.readFileSync(path.resolve(value));
```

Comment:
Shouldn’t this have utf8 encoding as well?

Reply:
@alan-agius4 hm, maybe, can you provide example?

Reply:
No need 
https://github.com/nodejs/node/blob/master/lib/fs.js#L392

`buffer.toString` :
```
encoding <string> The character encoding to use. Default: 'utf8'.

buf.toString([encoding[, start[, end]]])
```

Reply:
So default is `utf8`

Reply:
thanks for clarifying